src/js/ia/Helpers.js

```diff
@@ -30,7 +30,7 @@
         var timestring = full? " days ago" : "d";
         if (date) {
             // expected date format: YYYY-MM-DDTHH:mm:ssZ e.g. 2011-04-22T13:33:48Z
-            date = date.replace("/T.*Z/", "");
+            date = date.replace(/T.*Z/, "");
```

Comment:
Can momentjs use the date string as-is?  I found this in the docs http://momentjs.com/docs/#/parsing/special-formats/